hazelcast/src/main/java/com/hazelcast/map/impl/SizeEstimator.java

```diff
@@ -23,11 +23,17 @@
  */
 public interface SizeEstimator<T> {
 
+    /**
+     * Assume compressed object reference is used.
```

Comment:
Existing code. Subject of different discussion. Only made it more visible by moving it here.